src/redux/missions/missions.js

```diff
@@ -0,0 +1,51 @@
+/*
+eslint no-param-reassign: ["error", { "props": false }]
+--------------
+linter disable reason:
+According to the Redux Toolkit documentation state parameter should be reassigned.
+more info in: https://redux-toolkit.js.org/api/createSlice
+*/
```

Comment:
- [ ] Please remove all commented (unused) code to make your js modules more clean and readable.